tools/wpt/revlist.py

```diff
@@ -0,0 +1,299 @@
+import argparse
+import calendar
+import logging
+import os
+import subprocess
+import sys
+import time
+
+here = os.path.dirname(__file__)
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+logger = logging.getLogger()
+
+DEFAULT_EPOCH_STEP = 86400
+DEFAULT_EPOCH_THESHOLD = 600
+DEFAULT_EPOCH_UNTIL = calendar.timegm(time.gmtime())
+DEFAULT_HEAD_REVISION = "origin/master"
```

Comment:
It looks like now the `DEFAULT_HEAD_REVISION` and `--head-revision` argument is only needed because `git rev-list` is used internally and needs a commit.

By using `git for-each-ref` we can drop this requirement and simplify the code. I'll suggest that below.